### PR TITLE
Fix missing service definition

### PR DESCRIPTION
**Pull request by ylastapis**
| Q               | A
| --------------- | ---
| Bug fix?        |yes |
| New feature?    | no |
| BC breaks?      | no|

Fixes missing service definition, as moving to SF3 AND variable is already available through constructor

### CODE REVIEW

**Comment by hmert** [comment]
@ylastapis could you fix the conflict?

**Comment by ylastapis** [comment]
@hmert I'd like to, but I can't open https://github.com/APY/APYDataGridBundle/pull/904/conflicts.
keeps loading in the void...

**Comment by DonCallisto** [comment]
@ylastapis rebase your local branch with `dev-symfony3` of this bundle, during rebasing operations you will noticed with conflicts: resolve them by keeping all the things in our branch and modify just what you have highlighted here, then add the file to stage, continue the rebase until it is finished and `push -f` it onto your remote repository: this will fix every conflict.

**Comment by hmert** [comment]
@ylastapis done

File: Grid/Grid.php [hunk 1]
```diff
@@ -614,6 +614,7 @@ protected function processMassActions($actionId)
 
                 $this->prepare();
 
+
```
Comment (DonCallisto) [inline Grid/Grid.php]: Why is this space necessary?

File: Grid/Grid.php [hunk 2]
```diff
@@ -632,7 +633,7 @@ protected function processMassActions($actionId)
                         $action->getParameters()
                     );
 
-                    $subRequest = $this->container->get('request')->duplicate([], null, $path);
+                    $subRequest = $this->request->duplicate(array(), null, $path);
```
Comment (DonCallisto) [inline Grid/Grid.php]: This sounds good but we have a kind of mix in the code: sometimes we use `$this->{sometingRetrievedFromContainer}` and sometimes we use `$this->container->get({serviceName})` like line 638.

Don't know if is worth to keep same logic and initialize private members at construct or not. 

Any opinions?

Comment (ylastapis) [inline Grid/Grid.php]: Don't know about best practices on that, sry.
As ```$this->request``` was used everywhere and ```$this->container->get``` only here, I choosed the first option.

Comment (DonCallisto) [inline Grid/Grid.php]: @ylastapis yes, you've done right, I was asking to other members it it's the case to uniform every call that use `container`

**Pull request merged**